templates/CRM/Contact/Form/Edit/TagsAndGroups.tpl

```diff
@@ -12,24 +12,24 @@
   <div class="crm-accordion-header">{$title}</div>
   <div class="crm-accordion-body" id="tagGroup">
 {/if}
-    <table class="form-layout-compressed{if $context EQ 'profile'} crm-profile-tagsandgroups{/if}">
+    <table class="form-layout-compressed{if isset($context) && $context EQ 'profile'} crm-profile-tagsandgroups{/if}">
       <tr>
-        {if !$type || $type eq 'tag'}
+        {if empty($type) || (isset($type) && $type eq 'tag')}
```

Comment:
The second guard is redundant because after the `||` we know type is not empty and therefore isset.
```suggestion
        {if empty($type) || $type eq 'tag'}
```